### PR TITLE
Add information about site-status / publishing to README

### PR DESCRIPTION
**Pull request by alamb**
I found out about https://infra-reports.apache.org/#sitesource from @timsaucer : https://github.com/apache/datafusion-site/pull/34#issuecomment-2487010402

Let's add that to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,13 +72,18 @@ git checkout asf-site
...
+git push 
 ```
 
 #### Make PR, targeting the `asf-site` branch
 For example, see https://github.com/apache/datafusion-site/pull/9
 
+#### Check site status
+
+The website is updated from the `asf-site` branch on a schedule, so it may take
+some time for the  changes to be reflected. You can check the status at
```
Comment (timsaucer) [inline README.md:88]: I think I was wrong about the schedule. It seems to check on a schedule but also responds to pushes.

Comment (alamb) [inline README.md:88]:
```suggestion
The website is updated from the `asf-site` branch. You can check the status at 
```

Comment (alamb) [reply README.md:88]: Yes, I see now it was a temporary bug. I will revise the wording

**Comment by alamb** [comment]
Thank you for the review @timsaucer

**Pull request merged**